kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/AdaptiveBigQueryWriter.java

```diff
@@ -69,17 +69,17 @@ private boolean isTableMissingSchema(BigQueryException e) {
   }
 
   /**
-   * Sends the request to BigQuery, then checks the response to see if any errors have occurred. If
```

Comment:
We definitely don't want to delete this. This documentation blurb is specific to AdaptiveBigQueryWriter.performWriteRequest, not performWriteRequest in general.

So we don't have to repeat a bunch of stuff, maybe instead of individually documenting each @param, maybe just do a @see link to the abstract method in the BQWriter base class.

Reply:
make a lot of sense